CONTRIBUTING.md

```diff
@@ -30,11 +30,21 @@ For Git commit messages, our principle is that `git log --pretty=oneline` should
 - Does your PR include appropriate tests?
 - Make sure your PR title and description makes it easy for other developers to understand what the contained commits do. The title should say what the changes do. The description should expand on what it does (if not obvious from the title alone), and say why it is being done.
 - If your PR corresponds to an issue, add “Fixes #XX” to your pull request description. This will auto-close the corresponding issue when the commit is merged into master and tie the PR to the issue.
-- If your PR includes user-facing changes, you must add a line describing the change to [unreleased.rst](unreleased.rst) as part of your PR. Each entry in this document must start with the component to which is belongs, as in the following example:
+- If your PR includes user-facing changes, one or more commits in the PR must include in their body with a section between the ``CHANGELOG_BEGIN`` and ``CHANGELOG_END`` tags that includes relevant changelog entries, where each entry starts with the component to which is belongs in square brackets. Use RST to format links as this text will be added to the changelog upon release.
+
+The following is an example of a well-formed commit, including the description (first line) and a body that includes changelog additions:
+
+      Fixes #1311
+
+      Also fixes a typo in the Scala bindings documentation.
+
+      CHANGELOG_BEGIN
 
       - [Sandbox] Introduced a new API for package management.
         See `#1311 <https://github.com/digital-asset/daml/issues/1311>`__.
 
+      CHANGELOG_END
+
 ## Working with issues
```

Comment:
```suggestion
- In the unlikely event, you need to modify or remove a changelog entry made in a previously merged PRs before the relase, one or more commits in the PR must include in their body with a section between the ``CHANGELOG_BEGIN`` and ``CHANGELOG_END`` tags that includes a line of the form ``** Delete/Recplace Changelog from #xxxx about [Component] **``.

For instance in case of reverting a previous PR you could add a message like: 

      Revert #1322

      CHANGELOG_BEGIN
       
      ** Delete Changelog from #1322 about [Sanbox] **

      CHANGELOG_END

Or if you want to correct a changelog attached to a previously merged PR, you may use a message like:

      Fixes Changelog of #1322

      CHANGELOG_BEGIN

        ** Replace Changelog from #1322 about [Sanbox] **

      - [Sandbox] Introduced a new API for package management.
        See `#1312 <https://github.com/digital-asset/daml/issues/1312>`__.

      CHANGELOG_END

## Working with issues
```

Reply:
Addressed by https://github.com/digital-asset/daml/pull/3547/commits/5b06f4be6e791d550d0aaad0016e0dde661903c7